src/lj_err.c

```diff
@@ -384,11 +385,15 @@ typedef struct UndocumentedDispatcherContext {
   ULONG ScopeIndex;
   ULONG Fill0;
 } UndocumentedDispatcherContext;
+#else
+typedef void* UndocumentedDispatcherContext;
+#endif
 
 /* Another wild guess. */
-extern void __DestructExceptionObject(EXCEPTION_RECORD *rec, int nothrow);
+extern void LJ_CDECL __DestructExceptionObject(EXCEPTION_RECORD *rec,
+					       int nothrow);
```

Comment:
I want to say that people go to such lengths, but it has been too long since I've been in such codebases, so I'll concede the point.